ako-gateway-api/status/status.go

```diff
@@ -60,6 +60,10 @@ func DequeueStatus(objIntf interface{}) error {
 		utils.AviLog.Debugf("key: %s, msg: unknown object received", option.Key)
 		return nil
 	}
+	if option.Options.ServiceMetadata.HTTPRoute != "" {
```

Comment:
Changes here looks good for now as we don't have HTTPRoute status related changes here (landing from rest layer). We need to keep this in mind so that if in future we had such changes, we should move this to update and delete function of gateway_status.go file.

Reply:
Sure.Noted